### PR TITLE
Adding combined id generation to support version 2

### PR DESCRIPTION
**Pull request by djw8605**

### CODE REVIEW

**Comment by djw8605** [comment]
@jthiltges I believe this is what we discussed before.

**Comment by djw8605** [comment]
Great catch.  Fixed the timestamp change.

**Comment by djw8605** [comment]
Isn't the `data['timestamp']` an `int` though?

File: collector.py [hunk 1]
```diff
@@ -220,6 +221,31 @@ def get_RMQ_connection_parameters(self):
...
+        - source
+        - dest
+        - test type
+        """
+        if 'version' in message and message['version'] == "2": # Should we use a semvar library?
+            sha1_hash = hashlib.sha1()
+            sha1_hash.update(message['meta']['source'].encode('utf-8'))
+            sha1_hash.update(message['meta']['destination'].encode('utf-8'))
+            sha1_hash.update(self.TOPIC.encode('utf-8'))
+            sha1_hash.update(str(timestamp).encode('utf-8'))
```
Comment (jthiltges) [inline collector.py:243]: The timestamp is cast to string both places it's used. Makes the `calculateId()` call look a little cleaner.

Comment (djw8605) [reply collector.py:243]: Ah, ok.  Good point.

**Pull request merged**